index.php

```diff
@@ -9,11 +9,11 @@
 
 use microman\Form;
 use microman\FormBlueprint;
-use Kirby\Http\Server;
+use Kirby\Cms\App as Kirby;
 
-\Kirby\Cms\App::plugin('microman/formblock', [
+Kirby::plugin('microman/formblock', [
     'options' => [
-        'from_email' => 'no-reply@' . Server::host(),
+        'from_email' => 'no-reply@' . Kirby::instance()->environment()->host(),
```

Comment:
@afbora Have you tested that this does not break Kirby's loading order? Calling `App::instance()` from a plugin `index.php` outside of a callback is risky.

Reply:
Also I've tested this changes, seems works for me both: `Kirby::instance()->environment()->host()` and `(new Environment())->host()`

Reply:
You are right, should be fine because the environment is always loaded before plugins.